content/docs/components-and-props.md

```diff
@@ -233,30 +242,30 @@ function Comment(props) {
 
 [](codepen://components-and-props/extracting-components-continued)
 
-Extracting components might seem like grunt work at first, but having a palette of reusable components pays off in larger apps. A good rule of thumb is that if a part of your UI is used several times (`Button`, `Panel`, `Avatar`), or is complex enough on its own (`App`, `FeedStory`, `Comment`), it is a good candidate to be a reusable component.
+Till en början kan extrahering av komponenter verka tråkigt, men att ha ett antal återanvändbara komponenter lönar sig i slutändan för större applikationer. I allmänhet, om delar av ditt användargränssnitt används flera gånger (`Button`, `Panel`, `Avatar`) eller om det är tillräckligt komplex i sig själv (`App`, `FeedStory`, `Comment`), är det en bra kandidat för en återanvändbar komponent.
 
-## Props are Read-Only {#props-are-read-only}
+## Props är skrivskyddad {#props-are-read-only}
 
-Whether you declare a component [as a function or a class](#function-and-class-components), it must never modify its own props. Consider this `sum` function:
+Oavsett om du deklarerar en komponent [som en funktion eller en klass](#function-and-class-components), får den aldrig ändra sina egna Props. Ta denna `sum` funktion:
 
 ```js
 function sum(a, b) {
   return a + b;
 }
 ```
 
-Such functions are called ["pure"](https://en.wikipedia.org/wiki/Pure_function) because they do not attempt to change their inputs, and always return the same result for the same inputs.
+Sådana funktioner kallas ["ren"](https://en.wikipedia.org/wiki/Pure_function) eftersom de inte ändrar sina inputs och alltid returnerar samma resultat för samma inputs.
```

Comment:
Plural and singular mix

Reply:
I would say `Sådana funktioner kallas för ["rena"]` as a more proper translation.

Reply:
Fixing. 👍